source/settings/telemetry.txt

```diff
@@ -0,0 +1,132 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| tracks the following data:
+
+- The type of MongoDB deployment |compass-short| is connected to. For
+  example, Enterprise Edition, Community Edition, or Atlas Data Lake.
+
+- User actions performed in |compass-short|, such as creating
+  collections and indexes. |compass-short| only tracks the general
+  action performed, and does not track field values or namespaces.
+
+- Errors.
+
+|compass-short| *does not* track:
+
+- IP addresses, hostnames, usernames, or credentials.
+
+- Queries run in |compass-short|.
+
+- Data stored in your MongoDB deployment.
+
+- Personal identifiable information.
+
+For more information, see `MongoDB's Privacy Policy
+<https://www.mongodb.com/legal/privacy-policy?tck=docs_mongosh>`__.
+
+.. tip::
+
+   |compass-short| Isolated Edition automatically disables telemetry.
+   For more information on different |compass-short| editions, see
+   :ref:`compass-feature-table`.
+
+Steps
+-----
+
+You can toggle telemetry in either: 
+
+- The |compass-short| :ref:`Settings panel <compass-settings-reference>`
```

Comment:
[question] is there a reason why this is called the Settings panel in this line but the Settings window below? Is there a way to make this consistent?

```suggestion
- The |compass-short| :ref:`Settings panel <compass-settings-reference>`
```

Reply:
Standardized to "window".